Failed to activate the linter-elm-make package

Hi! Today i've updated version and got this error. It's on my side or something wrong with package?

**Atom Version**: 1.10.2
**Electron Version**: 0.37.8
**System**: Ubuntu 16.04.1
**Thrown From**: [linter-elm-make](https://github.com/mybuddymichael/linter-elm-make) package, v0.17.5

<details>
### Stack Trace

Failed to activate the linter-elm-make package

```
At Cannot find module 'diff-match-patch'

Error: Cannot find module 'diff-match-patch'
    at Module._resolveFilename (module.js:339:15)
    at Function.Module._resolveFilename (/opt/atom/resources/app.asar/src/module-cache.js:383:52)
    at Function.Module._load (module.js:290:25)
    at Module.require (module.js:367:17)
    at require (/opt/atom/resources/app.asar/src/native-compile-cache.js:50:27)
    at Object.<anonymous> (/home/anronin/.atom/packages/linter-elm-make/lib/linter-elm-make.js:13:34)
    at Module._compile (/opt/atom/resources/app.asar/src/native-compile-cache.js:103:30)
    at Object.defineProperty.value [as .js] (/opt/atom/resources/app.asar/src/compile-cache.js:208:21)
    at Module.load (module.js:357:32)
    at Function.Module._load (module.js:314:12)
    at Module.require (module.js:367:17)
    at require (/opt/atom/resources/app.asar/src/native-compile-cache.js:50:27)
    at Package.module.exports.Package.requireMainModule (/opt/atom/resources/app.asar/src/package.js:718:27)
    at Package.module.exports.Package.activateNow (/opt/atom/resources/app.asar/src/package.js:173:16)
    at /opt/atom/resources/app.asar/src/package.js:156:32
    at Package.module.exports.Package.measure (/opt/atom/resources/app.asar/src/package.js:92:15)
    at /opt/atom/resources/app.asar/src/package.js:149:26
    at Package.module.exports.Package.activate (/opt/atom/resources/app.asar/src/package.js:146:34)
    at PackageManager.module.exports.PackageManager.activatePackage (/opt/atom/resources/app.asar/src/package-manager.js:538:21)
    at /opt/atom/resources/app.asar/src/package-manager.js:519:29
    at Config.module.exports.Config.transactAsync (/opt/atom/resources/app.asar/src/config.js:337:18)
    at PackageManager.module.exports.PackageManager.activatePackages (/opt/atom/resources/app.asar/src/package-manager.js:514:19)
    at PackageManager.module.exports.PackageManager.activate (/opt/atom/resources/app.asar/src/package-manager.js:497:46)
    at /opt/atom/resources/app.asar/src/atom-environment.js:765:28
```

### Config

```json
{
  "linter-elm-make": {
    "lintOnTheFly": true
  }
}
```
</details>

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,4 +1,7 @@
 ## 0.17.5
+* Fix \#93.
+
+## 0.17.5
 * Fix `elm-format` integration bug.
 
 ## 0.17.4

--- a/lib/linter-elm-make.js
+++ b/lib/linter-elm-make.js
@@ -11,7 +11,6 @@ import _ from 'underscore';
 import chokidar from 'chokidar';
 import readDir from 'readdir';
 import Queue from 'promise-queue';
-import DiffMatchPatch from 'diff-match-patch';
 const atomLinter = require('atom-linter');
 
 export default {

